lib/Pod/Simple/Wiki/Confluence.pm

```diff
@@ -174,7 +174,10 @@ sub _end_L {
 
     # Handle links that are parsed as Pod links.
     if ( defined $link_section ) {
-        $link_target = $link_section;
+        # Other than for L<Link Text>, having a link section defined means
+        # that this is a page anchor link.
+        $link_target = ( $link_attrs->{raw} eq "$link_section" ) ? '' : '#';
```

Comment:
In general I avoid complex ternary assignments for maintenance reasons. Simpler ones are okay. In this case I would use something like:

```perl
if ( defined $link_attrs->{raw} and $link_attrs->{raw} ne $link_section ) {
    $link_target = '#';
}
```
Also, this is the source of the [failing tests](https://travis-ci.org/jmcnamara/pod-simple-wiki/builds/170000985).